configure/RULES.ioc.ISIS

```diff
@@ -2,7 +2,8 @@
 # may want to consider replacing $(ARCH) with %%EPICS_HOST_ARCH%% at some point,
 # but would need to adjust the generating of dllPath.bat too
 # note: runIOC.bat does not directly depend on dllCopy.bat but convenient to generate it this way
```

Comment:
Is this comment no longer seems relevant? I'm not sure if it's this change specifically that caused this but can we tidy it up as part of this?

Reply:
`dllPath.bat` is a target of every iocBoot Makefile, we would just need to remove it from all of these, I didn't do that as part of this ticket but it could be scripted. The comment lines above could indeed be removed, however related to the $(ARCH) comment I did wonder whether to change `dllPath.bat` to a generic file to call `dllPath-%EPICS_HOST_ARCH%.bat` I hadn't quite thought through which was best, so didn't want to remove `dllPath.bat` from every ioc Makefile to then decide to re-add it again later

Reply:
Ok, in the interests of getting this merged but making sure comments are valid and we don't forget about tidying up tasks could you:
* Update the comment to match what is in the codebase currently
* Create a ticket for removing all the references to `dllPath.bat` in the makefiles

Reply:
I put the ticket in as I know Tom wasted some time having built a debug epics by setting `epics_host_arch` but then the `dllpath` from runioc set the wrong distribution which confused his debugging. I created dllPath-arch as that was easiest and what the epics gateway does, but i'd also like to do something that epics base might adopt. Maybe i need to run the options  by epics-core:
- only create dllPath-arch (non backward compatible)
- create dllPath but explicitly use %EPICS_HOST_ARCH% in this file
- create dllPath and dllPath-arch   and have   dllPath call dllPath-arch

Reply:
Ok, I'm happy to merge this then, thank you.